spec/controllers/calls_controller_spec.rb

```diff
@@ -2,11 +2,49 @@
 
 describe CallsController,  type: :controller do
 
-  def setup_active_call_double(target_legislators:[])
+  def setup_active_call_double(target_legislators:[], new_record:false)
```

Comment:
Assignment Branch Condition size for setup_active_call_double is too high. [23.17/15]